Makefile

```diff
@@ -11,6 +11,14 @@ CFLAGS = -std=gnu99 -O2 -Wall -Wextra
 CFLAGS += -Wno-unused-label
 CFLAGS += -include src/common.h
 
+ENABLE_VALGRIND ?= 0
+ifeq ("$(ENABLE_UBSAN)", "1")
```

Comment:
It looks like a typo to me. Did you mean 'ENABLE_VALGRIND' instead of 'ENABLE_UBSAN'?

Reply:
Using `ENABLE_UBSAN` will enable `-g`, so will drop `ENABLE_VALGRIND` for now!